wiki/Abandoned_wasteland/id.md

```diff
@@ -0,0 +1,12 @@
+---
+stub: true
+tags:
+  - forum
+  - wastelanded
+---
+
+# Abandoned wasteland
+
+**Abandoned wasteland** adalah sebuah [forum-sampingan](https://osu.ppy.sh/community/forums)<!-- TODO --> yang bersifat pribadi digunakan untuk menyembunyikan forum-forum<!-- TODO: link --> dan forum topik<!-- TODO: link --> agar tidak terlihat oleh publik. Forum-sampingan ini hanya bisa diakses oleh para moderator<!-- TODO: link --> yang mempunyai kemampuan untuk memodifikasi forum. Perintah kepada para moderator mengenai hal tersebut adalah: `Taruh hal-hal yang tidak kita butuhkan kedalam sini.`.
+
+Forum-forum dan topik-topik disebut sebagai "wastelanded" ketika mereka dipindahkan ke dalam forum ini.
```

Comment:
```suggestion
Forum-forum dan topik-topik disebut sebagai "terbengkalai" ketika mereka dipindahkan ke dalam forum ini.
```

Reply:
i think we should follow the original term